webapp/src/components/Snackbar/Snackbar.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useRef} from 'react';
+import {useDispatch} from 'react-redux';
+
+import {Button} from '@brightscout/mattermost-ui-library';
+
+import {Icon} from 'components/Icon';
+
+import {getSnackbarState} from 'selectors';
+
+import usePluginApi from 'hooks/usePluginApi';
+import {closeAlert} from 'reducers/snackbar';
+import {alertTimeout} from 'constants/common.constants';
+
+import {SnackbarColor} from './Snackbar.types';
+
+export const Snackbar = () => {
+    const dispatch = useDispatch();
+    const {state} = usePluginApi();
+    const timeId = useRef(0);
+    const {isOpen, message, severity} = getSnackbarState(state);
+
+    const handleClose = () => dispatch(closeAlert());
+
+    useEffect(() => {
+        if (isOpen) {
+            timeId.current = window.setTimeout(() => {
+                // Hide the snackbar after 4 seconds
+                handleClose();
+            }, alertTimeout);
+        }
+
+        return () => {
+            clearTimeout(timeId.current);
+        };
+    }, [isOpen]);
+
+    const snackbarColorMap: Record<SnackbarColor, string> = {
+        error: 'var(--error-text)',
+        default: 'var(--center-channel-color)',
+        success: 'var(--online-indicator)',
+    };
```

Comment:
Are these colors not added to tw theme? 

If not, can we control this with classes (perhaps with modifier on `msteams-sync-rhs__snackbar`)? Or in any other way remove the inline styles

Reply:
These variables are defined in mattermost webapp

Reply:
You can add classes that use and add the class modifiers conditionally based on props